### ISSUE TITLE
Tabs not automatically progressing in Intermediate

### ISSUE THREAD

**Issue by loristokes** [inkdlab/writing-game#409]
When I complete tab 1, I have to click tab 2. Not an emergency--we can send this to Julie with this issue outstanding. But keep it in the queue for later.

**Comment by JSmith92** [comment]
Yes, we may not have this completed for Julie this morning, but it will be working soon.
